From 54f3edf2e98d367b0ab44d04c0f2408e6cb1b203 Mon Sep 17 00:00:00 2001
From: Pankaj Dev <pankaj.dev@st.com>
Date: Fri, 26 Jan 2024 12:19:14 +0530
Subject: [PATCH] arm64: defconfig: Enable ONBOARD USB Hub driver

USB Hub mounted on multiple stm32mp25 boards like eval, dk
Enabling driver for onboard-usb-hub

Change-Id: I0fa705ed99786c2de1cc5c74a594744aad975da0
Signed-off-by: Pankaj Dev <pankaj.dev@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/370798
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -97,6 +97,7 @@ CONFIG_TOUCHSCREEN_ILI210X=y
 CONFIG_UBIFS_FS=y
 CONFIG_USB_DWC3_STM32=y
 CONFIG_USB_HIDDEV=y
+CONFIG_USB_ONBOARD_HUB=y
 CONFIG_USB_STM32_USBH=y
 CONFIG_VIDEO_IMX335=m
 CONFIG_VIDEO_STM32_CSI=m
